sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -8,18 +8,20 @@
 #include "http/http.hpp"
```

Comment:
For the record, i didn't review this file

Reply:
For the record, you need to review this file T_T.

Reply:
That's 3300 lines of code change.

Reply:
Chrome is even slowing down and lagging when reviewing this file on GitHub.

Reply:
I skimmed it. I think it's fine.